float  confidence 
uint32 area      
float  x         
float  y          
float  z          
float  pixel_x    
float  pixel_y    
